corehq/apps/style/templates/style/includes/analytics_all.html

```diff
@@ -3,13 +3,10 @@
 {% load compress %}
 {% load hq_shared_tags %}
 
-<!DOCTYPE html>
```

Comment:
this is a partial template in other pages and so this is just showing up in the middle of the page.